dashboard/test/ui/features/star_labs/sharepage_logo.feature

```diff
@@ -59,7 +59,6 @@ Feature: Lab share page logo
     And I navigate to the share URL
     And I wait to see "#runButton"
     And I am on "http://studio.code.org/users/sign_out"
-    And I reload the page
```

Comment:
In `applab_project.feature` and `gamelab_project.feature`, both include going to sign-out page without this step so removing for now.